docs/src/how-to/nix-ld-nvidia-acceleration.md

```diff
@@ -0,0 +1,45 @@
+# How to use NVIDIA acceleration with nix-ld
+
+By default, the NixOS WSL configuration provides a symlink farm from
+/usr/lib/wsl/lib to the OpenGL packages. However, if you want to use the GPU
+with a non-NixOS compiled program via nix-ld (e.g. nvidia-smi), you also need to
+add the symlinks to the to the nix-ld libraries.
+
+```nix
+let
+  wsl-lib = pkgs.runCommand "wsl-lib" { } ''
```

Comment:
This is literally a copy of https://github.com/nix-community/NixOS-WSL/blob/adb6bc4b661a43328752b4575be4968a4990c033/modules/wsl-distro.nix#L84-L105 , right? How can we re-use those information? No one is going to keep those in sync. Maybe we should move the package to a read-only internal option?

Reply:
Yes, it's the same, we could make it more dynamic with a for loop or depending on the API design of this project, add it as an option.